stock_by_warehouse/static/src/components/warehouse/warehouse_field.js

```diff
@@ -70,6 +73,7 @@ export class StockByWarehouseField extends TextField {
                 title: this.info.title,
                 lines: this.lines,
                 onClose: this.closePopover,
+                show: this.show,
```

Comment:
Remove this line

Reply:
Done, solved in https://github.com/Vauxoo/addons-vauxoo/pull/1651